defaults/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+prometheus_server:
+  alertmanager:
+    url: ~
+  checksum: sha256:4779d5cf08c50ed368a57b102ab3895e5e830d6b355ca4bfecf718a034a164e0
+  download_dest: /tmp
+  download_url: https://github.com/prometheus/prometheus/releases/download/v1.7.1/prometheus-1.7.1.linux-amd64.tar.gz
+  environment: dev
+  group: prometheus
+  opts: ~
+  path:
+    config: /home/prometheus/config
+    data: /var/lib/prometheus/db
```

Comment:
Can you move data and logs to the /home/prometheus directory? I am not so fussed about the run directory but generally we try to pull all directories into the dedicated service user if possible.